direct/src/tkpanels/ParticlePanel.py

```diff
@@ -2920,14 +2920,14 @@ def setRadius(radius, f = force):
 # Create demo in root window for testing.
 if __name__ == '__main__':
     try:
-        base
+        base  # type: ignore[has-type, used-before-def]
```

Comment:
Maybe we can just clean this up by importing ShowBaseGlobal and checking if it has a base attribute